W-10796831: Update docs (#111)
mulesoft/docs-dataweave: modules/ROOT/pages/dw-dates-functions-localdatetime.adoc

[%header, cols="1,3"]
|===
| Name | Description
| `parts` | `Number` values for `year`, `month`, `day`, `hour`, `minutes`, and
            `seconds` fields. Valid values are numbers between 1 and 12 for the
            `month`, 1 through 31 for the `day`, 0 through 23 for the `hour`,

>             0 through 59 for `minutes`, and 0 through 59 (not inclusive; the field accepts 59.99, for example) for `seconds` fields.

Reviewer: TODO: CHECK "not inclusive" here, too. #2 instance

>             0 through 59 for `minutes`, and 0 through 59 (not inclusive; the field accepts 59.99, for example) for `seconds` fields.

Reviewer: ```suggestion
            0 through 59 for `minutes`, and 0 through 59 (including decimals, such as 59.99) for `seconds` fields.
```
